pages/documents/DeveloperTools/FaaS/Developing/event-templates.md

```diff
@@ -121,3 +123,203 @@ Additionally, Functions offers a number of templates that are not connected to e
   </tr>
 </tbody>
 </table>
+
+## Event: "Chat Post Survey E-Mail Transcript"
```

Comment:
Instead of having a header for each event I would use a tabel with the following columns:

Event Name | Template Name | Template Description